galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -736,7 +736,7 @@ def alter_hostname_settings(
 
 def configure_dab_required_settings(settings: Dynaconf) -> Dict[str, Any]:
     dab_settings = get_dab_settings(
-        installed_apps=settings.INSTALLED_APPS,
+        installed_apps=settings.INSTALLED_APPS + ['ansible_base.jwt_consumer'],
```

Comment:
This is added so that the setting `ANSIBLE_BASE_JWT_MANAGED_ROLES` is populated, which is now the standard way of identifying "platform" roles that shouldn't be modified locally, @fosterseth is getting into this stuff now